CryptKeeperZK/crypt-keeper-extension
# fix: build extension

## Explanation

Currently, we need to keep `dist` folder with all the assets we need and due to `.gitignore` it's not possible to keep them synchronized.

As a solution I removed dist folder and add webpack configuration to keep `dist` synchronized. The only one thing you need is just run `npm run dev` or `npm run build` (depends on env).

Details are bellow:
- [x] Remove dist folder
- [x] Move manifest.json and popup.html to src folder
- [x] Add html webpack plugin for popup.html
- [x] Add copy webpack plugin to copy assets (logos, manifest)

## More Information

Fixes #70 

## Screenshots/Screencaps

N/A

## Manual Testing Steps

1. Run `npm run dev` or `npm run build`
2. Install the extension
3. Check if it's working

## Pre-Merge Checklist


### + If there are functional changes:


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
